docs/tutorials/google/advanced_calibration.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>"assuming Application Default Credentials are setup." -&gt; set up</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done (note this was taken from https://quantumai.google/cirq/tutorials/google/start).</p>

Reply:
Yeah I assumed this was a copy paste but good to have it fixed here at least